templates/security/cis.html

```diff
@@ -66,7 +66,7 @@ <h3>Audit with OpenSCAP</h3>
     <div class="col-6">
       <h3>Configure and apply CIS hardening rules in minutes</h3>
       <p>The compliance tooling has two objectives: it lets our customers harden their Ubuntu systems effortlessly and then quickly audit those systems against the published CIS Ubuntu benchmarks. The SCAP content for audit tooling that scans the system for compliance is CIS certified.</p>
-      <a href="https://www.youtube.com/watch?v=wyEX0eyoK88" class="p-button--neutral p-link--external">Watch the video</a>
+      <a href="https://www.youtube.com/watch?v=IOLQQZVYbbg" class="p-button--neutral p-link--external">Watch the video</a>
```

Comment:
I think we include it on any link that takes you out the site, but I'm not sure. What do you think @sowasred2012 ?

Reply:
I think it's OK in this case - Vanilla 3.0 intends to remove the external link icon and class, so when we upgrade it may disappear anyway (though there's talk of _not_ removing it from Vanilla after all).